### ISSUE TITLE
password reset

### ISSUE THREAD

**Issue by Bari1up** [Pixel-Open/prestashop-cloudflare-turnstile#8]
Other pages work except password reset page.  Following error

Please validate the security field.

Console shows following but don't know if it's related.

[Please validate the security field.](normal:1 The resource https://challenges.cloudflare.com/cdn-cgi/challenge-platform/h/b/cmg/1/bTdlx6mXt3%2FRh7ojiHMqeswlWs28vCAvsml7%2BHFWn6g%3D was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.)

Have you seen this before? Thanks

**Comment by thierry-laval** [comment]
Yes, I’ve seen this on my side as well. It causes issues with both password reset and login. I’m not sure it’s directly related to this module — it might be more of a Turnstile issue. I still need to investigate more thoroughly, but for now, I’m switching my clients back to another system. Thanks to those who found a solution.
